.github/workflows/dependabot.yml

```diff
@@ -1,5 +1,10 @@
 name: Merge dependabot PRs automatically
-on: pull_request_target
+on:
+  pull_request_target:
+    paths:
+      - '!rbs.gemspec'
+      - '!Gemfile'
+      - '!steep/Gemfile'
```

Comment:
The `*/Gemfile` pattern would work, but the `!` doesn't work.

Reply:
Here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Yeah, but the action didn't run with this PR. I expected to run it because this PR doesn't change none of the files in the `!` pattern. So, I guess the negative (`!`) pattern doesn't work without *positive* pattern.

Reply:
Adding `**/*` at the top of patterns?

Reply:
Ah the `if:` section at the job section blocked running the job.